HHS/OPRE-OPS
# adds Validation to Agreements review page

## What changed
- moves Review Agreement into its own folder
- better JSDocs
- starts test suite
- adds SimpleAlert component

## Issue

#1131 

## How to test

1. Create an Agreement with bare minimum properties with(or without ) budget lines. 
2. Select the agreement from the Agreement list page.
3. See Alert errors for Agreement properties needed to be completed.
4. See Send to Review button is disabled.
